src/lexer/lex.c

```diff
@@ -330,11 +332,15 @@ TokenType ttype_one_char(char c) {
     case '?':
         return TT_QMARK;
     default:
-        PRINT_ERROR("Token type for token '%c' not recognized", c);
-        return TT_NO_TOKEN;
+		if (isdigit(c)) {
+			return TT_LITERAL;
+		} else {
+			return TT_IDENTIFIER;
```

Comment:
do we know that everything not listed is an identifier?

Reply:
Great question. I think we might need to check for one character symbols that definitely aren't tokens or identifiers. I can only think of things like the `@` symbol that isn't either, so we might want to have error handling for that later